extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -52,21 +52,36 @@ require([
 		if (modalNavHtml) {
 			$deferred.resolve(modalNavHtml);
 		} else {
-			nirvana.sendRequest({
-				controller: 'CommunityPageSpecial',
-				method: 'getModalHeaderData',
-				format: 'json',
-				type: 'get'
-			})
-			.then(function (response) {
-				modalNavHtml = mustache.render(templates.modalHeader, response);
-				$deferred.resolve(modalNavHtml);
+			modalNavHtml = mustache.render(templates.modalHeader, {
+				allText: $.msg('communitypage-modal-tab-all'),
+				adminsText: $.msg('communitypage-modal-tab-admins'),
+				leaderboardText: $.msg('communitypage-modal-tab-leaderboard'),
+				allMembersCount: allMembersCount,
+				adminsCount: adminsCount,
 			});
+			$deferred.resolve(modalNavHtml);
 		}
 
 		return $deferred;
 	}
 
+	function updateModalHeader() {
+		var allCountElement = $('#allCount'),
+			adminsCountElement = $('#adminsCount');
+
+		if (typeof allMembersCount === 'undefined') {
+			allCountElement.html('');
+		} else {
+			allCountElement.html('(' + allMembersCount + ')');
```

Comment:
Since all these have no need to be HTML, you should default to using `.text()` as it is the safe output format.